### ISSUE TITLE
HTTP push fails with Error 500 

### ISSUE THREAD

**Issue by simonthum** [go-gitea/gitea#1085]
Hi all,

I am trying to upload an insanely oversized repo (ex SVN) using plain http to my freshly installed gitea 1.0.1. on windows, installed as a service. Smaller repos worked fine.

```
$ git push mine --all
Counting objects: 85538, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (19927/19927), done.
Writing objects: 100% (85538/85538), 1.99 GiB | 11.15 MiB/s, done.
Total 85538 (delta 69000), reused 79958 (delta 64546)
error: RPC failed; HTTP 500 curl 22 The requested URL returned error: 500 Internal Server Error
fatal: The remote end hung up unexpectedly
fatal: The remote end hung up unexpectedly
Everything up-to-date
```

http.log:

````
2017/02/28 13:06:12 [...routers/repo/http.go:385 serviceRPC()] [E] fail to serve RPC(receive-pack): exit status 128
````

Other logs did not show correlated events.

It seems I am hitting some POST size limit. Is it possible to adjust this limit?

(I know how to organize repos better and smaller in general...)

**Comment by lunny** [comment]
This is a known issue #218 and should be resolved by #1030 which has been merged to v1.1.

**Comment by simonthum** [comment]
Sorry I didn't find it, thanks a lot!
